pipelines/rbsa/tasks/project_loadshapes.py

```diff
@@ -0,0 +1,132 @@
+import json
+import logging
+import pandas as pd
+import numpy as np
+import datetime
+from generics import task as t
+from generics.file_type_enum import SupportedFileReadType
+
+logger = logging.getLogger('LCTK_APPLICATION_LOGGER')
+
+class ProjectLoadshapes(t.Task):
+    """ 
+    This class is used to group sites into 3 digit zip codes
+    """
+    def __init__(self, name):
+        super().__init__(self)
+        self.name = name
+
+        self.input_artifact_loadshapes = 'loadshapes.csv'
+        self.input_artifact_correlation_matrix = 'correlation_matrix.csv'
+
+        self.my_data_files = [
+            { 'name': self.input_artifact_loadshapes, 'read_type': SupportedFileReadType.DATA },
+            { 'name': self.input_artifact_correlation_matrix, 'read_type': SupportedFileReadType.DATA },
+        ] 
+
+        self.output_artifact_total_loadshapes = 'total_loadshapes.csv'
+        self.task_function = self._task
+
+        self.theat = 15
+        self.tcool = 25
```

Comment:
after the previous branch gets merged in, we can update this to pull from the new config.
Otherwise this branch looks GTG

Reply:
fixed, updating and merging